keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/memory/InMemoryResourceRepository.kt

```diff
@@ -47,7 +50,7 @@ class InMemoryResourceRepository(
       .singleOrNull()
       ?.also {
         resources.remove(it)
-        events.remove(it)
+        resourceEvents.remove(it)
```

Comment:
Should we remove application level events too?

Reply:
I thought about that, but wasn't 100% sure we weren't using application references elsewhere (meaning, those events might still be relevant even if all the resources were removed). Also, the only place this method is called is from `deleteByApplication`, which in turn is only called by `CombinedRepository.deleteResourcesByApplication`, which, in turn... is not called from anywhere.

I think this may have been a side-effect of the change Emily made from delete-by-application to be delete-by-delivery-config. We might be able to remove this method altogether...